Configuration/AlCa/python/autoCond.py

```diff
@@ -31,16 +31,16 @@
     'run2_data_relval'             : '123X_dataRun2_relval_v2',
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi'      : '123X_dataRun2_PromptLike_HI_v2',
-    # GlobalTag for Run3 HLT: it points to the online GT
-    'run3_hlt'                     : '123X_dataRun3_HLT_v4',
     # GlobalTag with fixed snapshot time for Run2 HLT RelVals: customizations to run with fixed L1 Menu
-    'run2_hlt_relval'              : '123X_dataRun2_HLT_relval_v2',
+    'run2_hlt_relval'              : '123X_dataRun2_HLT_relval_v1',
```

Comment:
Is this going back in the version number wanted?

Reply:
No, that should have stayed the v2, I'll fix it